docs/reference/migration/apis/assistance.asciidoc

```diff
@@ -6,8 +6,13 @@
 <titleabbrev>Migration assistance</titleabbrev>
 ++++
 
-The Migration Assistance API analyzes existing indices in the cluster and
-returns the information about indices that require some changes before the
+IMPORTANT: The Migration Assistance API is deprecated, use the
```

Comment:
If someone uses this API in 6.7, will it return correct information ?

Reply:
I've made a tweak to this to make sure the behavior aligns, but as of this PR, if used in 6.7, this API will correctly identify which indices need to be reindexed, including internal indices. It will only say that indices need to be "upgraded" if they were not properly upgraded before upgrading the cluster to 6 and need to be fixed using the Migration Upgrade API.